main_pullrequest.go

```diff
@@ -66,8 +66,10 @@ func processGitHubPullRequest(
 			if err != nil {
 				log.Errorf("failed to start pipeline for PR: %s", err)
 				// post a comment only if GitLab is supposed to start a pipeline
-				gitlabReplyErrorMsg := "Missing CI config file"
-				if !strings.Contains(err.Error(), gitlabReplyErrorMsg) {
+				gitlabReplyErrorMsgMissingCI := "Missing CI config file"
+				gitlabReplyErrorMsgNoJobsForPipeline := "No stages / jobs for this pipeline"
+				if !strings.Contains(err.Error(), gitlabReplyErrorMsgMissingCI) ||
+					!strings.Contains(err.Error(), gitlabReplyErrorMsgNoJobsForPipeline) {
```

Comment:
This starts looking like and a candidate for an array/hashmap or whatever Go provides.

Reply:
Does `regexp` look more elegant?